.github/workflows/ci.yml

```diff
@@ -22,38 +22,40 @@ jobs:
         uses: nrwl/nx-set-shas@v2
         with:
           main-branch-name: 'master'
-      - uses: actions/setup-node@v1 
-        with: 
+      - uses: actions/setup-node@v1
+        with:
           node-version: '16'
       - run: npm ci
-      - run: npx nx-cloud start-ci-run    
-      - run: npm run build
-      - run: npx nx affected --target=lint --parallel --max-parallel=3
-      - run: npx nx affected --target=test --parallel --max-parallel=3
-      - run: npm run e2e
-      - run: npx nx-cloud stop-all-agents
+      - run: yarn dlx nx-cloud start-ci-run
+      - run: yarn run build
+      - run: yarn dlx nx affected --target=lint --parallel --max-parallel=3
+      - run: yarn dlx nx affected --target=test --parallel=3
+      - run: yarn run e2e
+      - run: yarn dlx nx-cloud stop-all-agents
   pr:
     runs-on: ubuntu-latest
     if: ${{ github.event_name == 'pull_request' }}
     steps:
       - uses: actions/checkout@v2
         with:
-          ref: ${{ github.event.pull_request.head.ref }}
+          # see: https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/
+          # regarding `pull_request_target`
+          ref: ${{ github.event.pull_request_target.head.ref }}
```

Comment:
@srleecode please get this functionality into `master` so PRs from forks can work with the CI